enarx-keep/src/backend/kvm/vm/cpu.rs

```diff
@@ -7,11 +7,11 @@ use crate::backend::kvm::shim::{
     MemInfo, SYSCALL_TRIGGER_PORT, SYS_ENARX_BALLOON_MEMORY, SYS_ENARX_MEM_INFO,
 };
 use crate::backend::{Command, Thread};
+use crate::sallyport;
```

Comment:
I'd prefer to stick with `use crate::sallyport::Reply`... Is there some reason for this change?

Reply:
no reason, changed `sallyport::Block` instead then